include/cpu/cpus.h

```diff
@@ -30,25 +29,26 @@ void cpu_stop_current(void);
 void cpu_synchronize_all_post_reset(void);
 void cpu_synchronize_all_post_init(void);
 
+typedef struct TimersStateScale {
+    int32_t clock_scale_enable;
```

Comment:
Not sure if we need enable. A factor of 1 should mean disable.

Reply:
If that's the case, why does it exist at all then? Maybe we can get rid of it?

Reply:
I got rid of it